pkg/proxy/tcp_proxy.go

```diff
@@ -8,12 +8,14 @@ import (
 	"github.com/Sirupsen/logrus"
 )
 
+// TCPProxy is a proxy for TCP connections.
```

Comment:
What is the role of a `TCPProxy`? When should it be used? As a resource, does it require management?